Common/File/FileUtil.cpp

```diff
@@ -258,6 +272,10 @@ static bool ResolvePathVista(const std::wstring &path, wchar_t *buf, DWORD bufSi
 	if (getFinalPathNameByHandleW) {
 #if PPSSPP_PLATFORM(UWP)
 		HANDLE hFile = CreateFile2FromAppW(path.c_str(), GENERIC_READ, FILE_SHARE_READ, OPEN_EXISTING, nullptr);
+#if !defined(__LIBRETRO__)
+		//Use UWP StorageManager to get handle
+		hFile = CreateFileUWP(path, GENERIC_READ, FILE_SHARE_READ, OPEN_EXISTING);
```

Comment:
So in this case, we first call CreateFile2FromAppW and then CreateFileUWP? Doesn't seem quite right, the CreateFile2FromAppW call should probably be moved into an #else here.